<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raggamuffin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabCourses">
     <attribute name="title">
      <string>courses</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>339</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelUnitMass">
         <property name="text">
          <string>Comment donner la quantité ?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="buttonUnit">
         <property name="text">
          <string>Unité</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="buttonMasse">
         <property name="text">
          <string>Masse (kg)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabFood">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>411</width>
        <height>451</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabLegumes">
       <attribute name="title">
        <string>Légumes</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabFruits">
       <attribute name="title">
        <string>Fruits</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabViandes">
       <attribute name="title">
        <string>Viandes</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>291</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableListeCourses"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Enregistrer la liste de courses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRemove">
         <property name="text">
          <string>Enlever l'aliment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonModify">
         <property name="text">
          <string>Modifier la quantité d'aliment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRAZ">
         <property name="text">
          <string>RAZ de la liste des courses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonValidate">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 0);</string>
         </property>
         <property name="text">
          <string>Valider les courses</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabNutri">
     <attribute name="title">
      <string>bilan nutritionnel</string>
     </attribute>
     <widget class="QTableWidget" name="tableListeCourses_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPhysio">
     <attribute name="title">
      <string>bilan physiologique</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
